src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/ExpandedMenuToggle.tsx

```diff
@@ -44,52 +48,61 @@ const text = ({ showExpandedMenu }: { showExpandedMenu: boolean }) => css`
         display: inline-block;
         height: 8px;
         margin-left: 6px;
-        transform: ${showExpandedMenu
-            ? 'translateY(1px) rotate(-135deg)'
-            : 'translateY(-3px) rotate(45deg)'};
+
+        /*
+            IMPORTANT NOTE:
+            we need to specify the adjacent path to the a (current) tag
+            to apply styles to the nested tabs due to the face we use ~
+            to support NoJS
+        */
+        transform: translateY(-3px) rotate(45deg);
+        ${`#${CHECKBOX_ID}:checked ~ & {
+            transform: translateY(1px) rotate(-135deg);
+        }`}
+
         transition: transform 250ms ease-out;
         vertical-align: middle;
         width: 8px;
     }
     :hover:after {
-        transform: ${showExpandedMenu
-            ? 'translateY(-2px) rotate(-135deg)'
-            : 'translateY(0) rotate(45deg)'};
+        /*
+            IMPORTANT NOTE:
+            we need to specify the adjacent path to the a (current) tag
+            to apply styles to the nested tabs due to the face we use ~
+            to support NoJS
+        */
+        transform: translateY(0) rotate(45deg);
+        ${`#${CHECKBOX_ID}:checked ~ & {
+            transform: translateY(-2px) rotate(-135deg);
+        }`}
     }
 `;
 
 interface Props {
     display: Display;
-    showExpandedMenu: boolean;
-    toggleExpandedMenu: (value: boolean) => void;
     ariaControls: string;
+    CHECKBOX_ID: string;
 }
 
 export const ExpandedMenuToggle = ({
     display,
-    toggleExpandedMenu,
     ariaControls,
-    showExpandedMenu,
+    CHECKBOX_ID,
 }: Props) => (
     <>
         <VeggieBurger
             display={display}
-            showExpandedMenu={showExpandedMenu}
-            toggleExpandedMenu={toggleExpandedMenu}
+            CHECKBOX_ID={CHECKBOX_ID}
             ariaControls={ariaControls}
             key="VeggieBurger"
         />
-        <button
+        <LabelButton
             className={openExpandedMenu(display)}
-            onClick={() => toggleExpandedMenu(!showExpandedMenu)}
-            aria-controls={ariaControls}
-            key="OpenExpandedMenuButton"
-            data-link-name={`nav2 : veggie-burger : ${
-                showExpandedMenu ? 'show' : 'hide'
-            }`}
+            ariaControls={ariaControls}
```

Comment:
Not sure if it helps, but Heydon Pickering has long advocates [ignoring aria-controls](https://heydonworks.com/article/aria-controls-is-poop/). Would removing aria-controls make the component easier to work with? If so it would be worth checking whether the advice in this article is still accurate.

Reply:
yeah I just read that, I think I will as a result remove it